jekyll/_cci2/workflows.md

```diff
@@ -523,6 +525,13 @@ When you use workflows, you increase your ability to rapidly respond to failures
 
 This section describes common problems and solutions for Workflows.
 
+### Workflow and Subsequent Jobs Do Not Trigger
+
+Oftentimes if you do not see your workflows triggering, a configuration error is
```

Comment:
```suggestion
If you do not see your workflows triggering, a common cause is a configuration error
```
```suggestion
Oftentimes if you do not see your workflows triggering, a configuration error is
```

Reply:
Just a suggestion as "oftentimes" isn't (I don't think) in common usage in the UK so thought it might not localize well. Take it or leave it :-D